controllers/storagecluster/initialization_reconciler_test.go

```diff
@@ -101,7 +102,7 @@ func createUpdateRuntimeObjects(cp *Platform) []runtime.Object {
 		updateRTObjects = append(updateRTObjects, cos)
 	}
 	// Create 'route' only for non-cloud platforms
-	if !avoidObjectStore(cp.platform) {
+	if !avoid {
```

Comment:
Combine all these into one `if` stanza. They all check the same thing with no logic in between.